hack/testing/test-001-operator-sdk-e2e.sh

```diff
@@ -57,25 +47,18 @@ cleanup(){
 }
 trap cleanup exit
 
-if oc get project ${TEST_NAMESPACE} > /dev/null 2>&1 ; then
-  echo using existing project ${TEST_NAMESPACE}
-else
-  oc create namespace ${TEST_NAMESPACE}
+if [ "${DO_SETUP:-true}" == "true" ] ; then
+  for item in "${TEST_NAMESPACE}" "openshift-operators-redhat"; do
+    if oc get project ${item} > /dev/null 2>&1 ; then
+      echo using existing project ${item}
+    else
+      oc create namespace ${item}
+    fi
+  done
+
+  deploy_elasticsearch_operator
 fi
 
 sed -i "s/namespace: openshift-logging/namespace: ${TEST_NAMESPACE}/g" ${manifest}
 
-oc create -n ${TEST_NAMESPACE} -f \
-https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml ||:
-oc create -n ${TEST_NAMESPACE} -f \
-https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml ||:
-
-TEST_NAMESPACE=${TEST_NAMESPACE} go test ./test/e2e/... \
-  -root=$(pwd) \
-  -kubeconfig=${KUBECONFIG} \
-  -globalMan manifests/04-crd.yaml \
-  -namespacedMan ${manifest} \
-  -v \
-  -parallel=1 \
-  -singleNamespace \
-  -timeout 1200s
+TEST_NAMESPACE=${TEST_NAMESPACE} go test ./test/e2e/.
```

Comment:
we don't still need the extended timeout? we bumped that because we saw the tests were hitting the default `600s` limit before

Reply:
@ewolinetz is that an arg available to `go test` or just an sdk flag?

Reply:
those were all for `go test`